### PR TITLE
fix(curriculum): external roman numeral links in JS projects

### PR DESCRIPTION
**Pull request by Sboonny**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

clear the 50th task in #40749

### CODE REVIEW

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/javascript-algorithms-and-data-structures-projects/roman-numeral-converter.md [hunk 1]
```diff
@@ -10,7 +10,24 @@ dashedName: roman-numeral-converter
...
+|L  |50      |
+|XL  |40      |
+|X  |10      |
+|IX  |0      |
+|V |5      |
+|IV |4      |
+|I |1      |
+
+
+All roman numerals answers should be provided in upper-case.
```
Comment (raisedadead) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/javascript-algorithms-and-data-structures-projects/roman-numeral-converter.md:30]:
```suggestion
| Roman numerals | Arabic numerals |
|----------------|-----------------|
| M              | 1000            |
| CM             | 900             |
| D              | 500             |
| CD             | 400             |
| C              | 100             |
| XC             | 90              |
| L              | 50              |
| XL             | 40              |
| X              | 10              |
| IX             | 0               |
| V              | 5               |
| IV             | 4               |
| I              | 1               |

All roman numerals answers should be provided in upper-case.
```

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/build-a-roman-numeral-converter-project/roman-numeral-converter.md [hunk 2]
```diff
@@ -10,7 +10,24 @@ dashedName: build-a-roman-numeral-converter
...
+|L  |50      |
+|XL  |40      |
+|X  |10      |
+|IX  |0      |
+|V |5      |
+|IV |4      |
+|I |1      |
+
+
+All roman numerals answers should be provided in upper-case.
```
Comment (raisedadead) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/build-a-roman-numeral-converter-project/roman-numeral-converter.md:30]:
```suggestion
| Roman numerals | Arabic numerals |
|----------------|-----------------|
| M              | 1000            |
| CM             | 900             |
| D              | 500             |
| CD             | 400             |
| C              | 100             |
| XC             | 90              |
| L              | 50              |
| XL             | 40              |
| X              | 10              |
| IX             | 0               |
| V              | 5               |
| IV             | 4               |
| I              | 1               |

All roman numerals answers should be provided in upper-case.
```

**Review by raisedadead — CHANGES_REQUESTED** [review]
Nitpicking on formating.

**Review by Sembauke — APPROVED** [review]

**Review by raisedadead — APPROVED** [review]

**Comment by WestonG40** [comment]
Co-authored-by: @NicholausJ  <email@example.com>

**Comment by WestonG40** [comment]
Co-authored-by: @WestonG40 <email@example.com>

**Pull request merged**